packages/widgets/src/Pagination/Pagination.styles.js

```diff
@@ -0,0 +1,45 @@
+import css from 'styled-jsx/css'
+import { colors, spacers } from '@dhis2/ui-constants'
+
+export default css`
+    .container {
+        display: flex;
+        justify-content: flex-end;
+        flex-wrap: wrap;
```

Comment:
The wrapping for smaller viewports seems a bit awkward. Maybe we could make this a bit nicer without resorting to media queries (since mq's don't seem like a good idea from a library perspective?):

<img width="453" alt="Screenshot 2020-09-08 at 12 11 22" src="https://user-images.githubusercontent.com/7355199/92463806-9abe3300-f1cc-11ea-843a-48efc6e7b386.png">

Reply:
🤔  hmmm thanks for identifying this issue. I had actually looked into this before I implemented Joe's input (all his comments are gone because I squashed my commits 😢 ), and it looked a lot better before (not great but at least better). When I've started aligning stuff on the right this must have gotten this way. I'll look into it.

Reply:
Fixed the alignment itself in eecd7b9 but this produced a problem when `PageSizeSelect` is not rendered (I guess that's why I was aligning at the flex-end). So I fixed that with a spacer div in ab1e6d5